website/extensions/placeholder.md

```diff
@@ -10,6 +10,16 @@ Show some placeholder text when the current text block is empty or the whole doc
 import { definePlaceholder } from 'prosekit/extensions/placeholder'
 
 const extension = definePlaceholder({ placeholder: 'Type Something...' })
+
+// or show dynamic placeholders by node
+const extension = definePlaceholder({
+  placeholder: (node: Node) => {
+    if (node.type.name === 'heading') {
+      return `Heading ${node.attrs.level}`
+    }
+    return 'Type something...'
+  },
+})
```

Comment:
This is a `ts twoslash` code block ([docs link](https://shiki.style/packages/vitepress#twoslash)) and typescript compiler will handle this code block, thus you cannot redeclare the variable `extension` here. You can add a new code block.

Reply:
A new code block has been added.